app/assets/javascripts/discourse/tests/integration/components/d-section-test.gjs

```diff
@@ -0,0 +1,19 @@
+import { module, test } from "qunit";
+import { setupRenderingTest } from "discourse/tests/helpers/component-test";
+import { render } from "@ember/test-helpers";
+import DSection from "discourse/components/d-section";
+
+module("Integration | Component | d-section", function (hooks) {
+  setupRenderingTest(hooks);
+
+  test("can set classes on the body element", async function (assert) {
+    await render(<template>
```

Comment:
We can wrap this test in `withSilencedDeprecationsAsync("discourse.d-section", ...)`, and then unsilence the deprecation in ember-cli-deprecation-workflow.

Reply:
I'd rather enable the deprecation after we have a chance to update plugins (and we'll do that after the next beta, I think)

Reply:
Ok sounds good :+1: